django/contrib/admin/sites.py

```diff
@@ -494,16 +494,14 @@ def _build_app_dict(self, request, label=None):
                     'models': [model_dict],
                 }
 
-        if label:
-            return app_dict.get(label)
         return app_dict
 
-    def get_app_list(self, request):
+    def get_app_list(self, request, app_label=None):
         """
         Return a sorted list of all the installed apps that have been
         registered in this site.
```

Comment:
The docstring should probably be updated so that it more accurately describes the new and improved function :)

Reply:
Although, having said that. If you made the suggested change below there would be no longer be a need to update this function.